app/views/shared/results/_search_results.html.haml

```diff
@@ -18,7 +18,7 @@
         with dates
       - if provider_facets.present?
         from
-        = "#{ 'over' if provider_facets.count >= 2000 }"
+        = "#{ 'over' if provider_facets.count >= 240 }"
```

Comment:
@AudreyAltman I did consider removing that entirely, but thought I should keep this PR within the scope of the ticket, which is only to address the denial-of-service vulnerabilities at the moment. It's always been off, for the past 4 or so years, because there's always been a mismatch between the number reported and the number of records on-screen, if the number was more than 300. For example, it might have said "400 subjects" while only showing 300, or "over 2,000 subjects" while only showing 300. Note how the decision to say "over" is based on the facet count set elsewhere in the code (in search.rb) and not with the number of pages of 30 records that are drawn by the view!

I could see changing the `240` count if people want. To really avoid changing the outward behavior I could have kept it at 10 pages of 30 instead of 8 pages of 30. I was just trying to cut down the number of results displayed (and the sizes of responses going over the wire) and I figured it was already an incomplete picture with the top 300. It was hard to resist cutting the fat even more, considering the display has been less than ideal to begin with.